### PR TITLE
Get rid of unused canvas2d test templates

### PR DESCRIPTION
**Pull request by mysteryDate**
Part of the effort described here: https://github.com/web-platform-tests/wpt/issues/22979

AFAICT the vast majority of templates in `html/canvas/tools/templates.yaml` are completely unused. I plan on adding a few templates to this, but it makes sense to clean up first unless these are somehow necessary in a way that I am missing.

### CODE REVIEW

**Comment by annevk** [comment]
That mochitest stuff is probably something from Firefox. However, given all the files moved I wonder if all that regressed anyway.

cc @jdashg @lsalzman

**Review by Ms2ger — APPROVED** [review]

File: html/canvas/tools/templates.yaml [hunk 1]
```diff
@@ -108,271 +27,4 @@ w3c: |
...
-
-results: |
-    <!DOCTYPE html>
-    <title>Canvas tests - results</title>
-    <link rel="stylesheet" href="../results.css">
-    <table>
-    <col id="col1">
-    <tr>
-    <th>Test
+    %(scripts)s%(images)s
```
Comment (Ms2ger) [inline html/canvas/tools/templates.yaml:30]: Restore the newline to fix update-built-tests

Comment (mysteryDate) [reply html/canvas/tools/templates.yaml]: Thanks! I was having a heck of a time hunting that down.

**Review by kdashg — APPROVED** [review]

**Comment by jdashg** [comment]
We can always revive these if we need them again.

**Pull request merged**